.github/CODEOWNERS

```diff
@@ -98,3 +98,6 @@
 /test/util                              @DataDog/agent-all
 
 /tools/ebpf/                            @DataDog/burrito
+
+# Match specific files first (at the bottom of the file to be matched first)
+config_template.yaml                    @DataDog/agent-all @DataDog/baklava
```

Comment:
Interesting, is that the recommended way to do this? Because right now we're not doing this at all for other specific files (eg. https://github.com/DataDog/datadog-agent/blob/aa3a1a8e42ceba17dc232e4362c27829a9a4cbe8/.github/CODEOWNERS#L88-L91
)

Is there an advantage to not putting the full path to the file?

Reply:
if the file is ever moved, the rule will still be valid. But not if the file name is changed obviously. Followed what's done on integrations-core: https://github.com/DataDog/integrations-core/blob/master/.github/CODEOWNERS (it's mandatory there since the rule has to match a bunch of files)

I can change it to the full absolute path if you prefer

Reply:
To be honest, I like our current codeowners structure because it's easy to see how the repo owners are structured per base folder. That's why I'd put it with at the end of the `/pkg/` block.

As for the full absolute path, I'd use it since here we're only matching one file, and I believe we don't expect this file to move often.

Reply:
ok, changed 👍